miio/airpurifier.py

```diff
@@ -110,6 +110,11 @@ def mode(self) -> OperationMode:
         """Current operation mode."""
         return OperationMode(self.data["mode"])
 
+    @property
+    def sleep_mode(self) -> Optional[OperationMode]:
+        """Operation mode of the sleep state. (Idle vs. Silent)"""
+        return OperationMode(self.data["sleep_mode"])
```

Comment:
Here's another optional, I'm unsure what happens if the sleep_mode is None, will it just raise an exception or will it return an OperationMode (which one?)?

Reply:
It will return "None" now.